README.md

```diff
@@ -26,14 +26,14 @@ You'll find many other useful documents on our [Wiki](https://github.com/istio/i
 
 ## Introduction
 
-Istio is an open platform for providing a uniform way to integrate
-microservices, manage traffic flow across microservices, enforce policies
+[Istio](https://istio.io/latest/docs/concepts/what-is-istio/) is an open platform for providing a uniform way to [integrate
+microservices](https://istio.io/latest/docs/examples/microservices-istio/), manage [traffic flow](https://istio.io/latest/docs/concepts/traffic-management/) across microservices, enforce policies
 and aggregate telemetry data. Istio's control plane provides an abstraction
 layer over the underlying cluster management platform, such as Kubernetes.
 
 Istio is composed of these components:
 
-- **Envoy** - Sidecar proxies per microservice to handle ingress/egress traffic
+- [Envoy](https://www.envoyproxy.io/learn/service-mesh) - [Sidecar proxies](https://istio.io/latest/docs/reference/config/networking/sidecar/) per microservice to handle ingress/egress traffic
```

Comment:
"Sidecar proxies" != Sidecar resource. The former is a deployment pattern and the latter a configuration resource. I would suggest dropping this?

Reply:
Done